library/ecp.c

```diff
@@ -551,6 +551,10 @@ static const mbedtls_ecp_curve_info ecp_supported_curves[] =
 #if defined(MBEDTLS_ECP_DP_CURVE25519_ENABLED)
     { MBEDTLS_ECP_DP_CURVE25519,   29,     256,    "x25519"            },
 #endif
+#if defined(MBEDTLS_ECP_DP_SM2P256V1_ENABLED)
+    /* https://tools.ietf.org/id/draft-yang-tls-tls13-sm-suites-05.html */
```

Comment:
Should presumably be updated to [RFC 8998](https://datatracker.ietf.org/doc/html/rfc8998)

Reply:
Indeed, thanks for the reference